source/braille.py

```diff
@@ -1124,11 +1177,50 @@ def getFormatFieldBraille(field, fieldCache, isAtStart, formatConfig):
 				# Translators: brailled when text contains a bookmark
 				text = _("bkmk")
 				textList.append(text)
+
+	if (
+		config.conf["braille"]["fontAttributeDisplay"].calculated() == FontAttributesBrailleModeFlag.TAGS
+		and (formattingTags := _getFormattingTags(field, fieldCache, formatConfig)) is not None
+	):
+		textList.append(formattingTags)
+
 	fieldCache.clear()
 	fieldCache.update(field)
 	return TEXT_SEPARATOR.join([x for x in textList if x])
 
 
+def _getFormattingTags(
+	field: dict[str, str], fieldCache: dict[str, str], formatConfig: dict[str, bool]
+) -> str | None:
+	"""
+	Get the formatting tags for the given field and cache.
+
+	Args:
+		field: The format field.
+		fieldCache (dict): The previous format field.
+		formatConfig: The user's format config.
+
+	Returns:
+		The formatting tag as a string, or None if no formatting is applied.
```

Comment:
please use sphinx syntax

```suggestion
	:param field: The format field.
	...
```